e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -134,22 +134,40 @@ Scenario('HMCTS admin enters hearing details and submits', async (I, caseViewPag
   I.seeAnswerInTab(4, 'Judge and legal advisor', 'Legal advisor\'s full name', hearingDetails[1].judgeAndLegalAdvisor.legalAdvisorName);
 });
 
-Scenario('HMCTS admin creates C21 order for the case', async (I, caseViewPage, createOrderEventPage) => {
+Scenario('HMCTS admin creates multiple orders for the case', async (I, caseViewPage, createOrderEventPage) => {
   await caseViewPage.goToNewActions(config.administrationActions.createOrder);
+  await createOrderEventPage.selectOrderType(orders[0].orderType);
+  await I.retryUntilExists(() => I.click('Continue'), '#order_orderTitle');
   await createOrderEventPage.enterC21OrderDetails();
   await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
   await createOrderEventPage.enterJudgeAndLegalAdvisor(orders[0].judgeAndLegalAdvisor.judgeLastName, orders[0].judgeAndLegalAdvisor.legalAdvisorName);
   await I.completeEvent('Save and continue');
-  const now = new Date();
+  const c21OrderTime = new Date();
```

Comment:
should this just be OrderTime and not c21?

Reply:
we need different time for each order, we are testing C21 + care order in this scenario, made sense to name the date for which order it was testing

Reply:
see slightly below: careOrderTime on line 162

Reply:
But we could just have an orderTime and update it, that's probably better

Reply:
yeah lets go with orderTime :)